### PR TITLE
LibJS: Add the TypedArray.of() method

### PR DESCRIPTION
**Pull request by IdanHo**
This fixes 20 test262 test cases.

### CODE REVIEW

File: Userland/Libraries/LibJS/Runtime/TypedArrayConstructor.cpp [hunk 1]
```diff
@@ -26,6 +27,8 @@ void TypedArrayConstructor::initialize(GlobalObject& global_object)
     define_property(vm.names.prototype, global_object.typed_array_prototype(), 0);
     define_property(vm.names.length, Value(0), Attribute::Configurable);
 
+    define_native_function(vm.names.of, of, 0, Attribute::Configurable);
```
Comment (linusg) [inline Userland/Libraries/LibJS/Runtime/TypedArrayConstructor.cpp:30]: That's not correct:

```suggestion
    u8 attr = Attribute::Writable | Attribute::Configurable;
    define_native_function(vm.names.of, of, 0, attr);
```

**Review by linusg — COMMENTED** [review]
Please add some basic in-tree tests for this :)

File: Userland/Libraries/LibJS/Runtime/TypedArrayConstructor.cpp (hunk 1 above)
Comment (IdanHo) [reply Userland/Libraries/LibJS/Runtime/TypedArrayConstructor.cpp]: my bad, fixed!

**Comment by IdanHo** [comment]
Tests added!

**Review by linusg — APPROVED** [review]

**Pull request merged**